packages/js/src/generators/library/utils/add-release-config.ts

```diff
@@ -132,30 +131,16 @@ export async function addReleaseConfigForNonTsSolution(
     },
   };
 
-  if (useLegacyVersioning) {
-    projectConfiguration.release = {
-      version: {
-        generatorOptions: {
-          packageRoot,
-          // using git tags to determine the current version is required here because
-          // the version in the package root is overridden with every build
-          currentVersionResolver: 'git-tag',
-          fallbackCurrentVersionResolver: 'disk',
-        },
-      },
-    };
-  } else {
-    // TODO: re-evaluate this config in new versions
-    projectConfiguration.release = {
-      version: {
-        manifestRootsToUpdate: [packageRoot],
-        // using git tags to determine the current version is required here because
-        // the version in the package root is overridden with every build
-        currentVersionResolver: 'git-tag',
-        fallbackCurrentVersionResolver: 'disk',
-      },
-    };
-  }
+  // TODO: re-evaluate this config in new versions
```

Comment:
I think we can strip this TODO, it's lost meaning now that a lot of time has passed and this code path (non TS solution) is already kind of legacy